config/arch/sh.in

```diff
@@ -5,6 +5,7 @@
 ## select ARCH_USE_MMU
 ## select ARCH_SUPPORTS_BOTH_ENDIAN
 ## select ARCH_DEFAULT_LE
+## select ARCH_REQUIRES_MULTILIB
```

Comment:
I'm not sure why we need to have `ARCH_REQUIRES_MULTILIB` when its sole
reason to exist is to select `MULTILIB`. Why not select `MULTILIB` directly?

Well, I can see that is is more meaning full to say "arch requires multilib" rather
than just "force multilib" on; it gives the reason. But still...

Anyway, that's not this patch that introduces it, so:

Reviewed-by: "Yann E. MORIN" <yann.morin.1998@free.fr>